src/routes/TomogramList.tsx

```diff
@@ -0,0 +1,131 @@
+import {
+  Divider,
+  Heading,
+  HStack,
+  Spacer,
+  Box,
+  Select,
+  Text,
+  Image,
+  Card,
+  CardBody,
+  Grid,
+  VStack,
+} from "@chakra-ui/react";
+import { useEffect } from "react";
+import { Link, useLoaderData } from "react-router";
+import { Pagination, DebouncedInput } from "@diamondlightsource/ui-components";
+import { usePaginationSearchParams } from "utils/hooks";
+import { Options } from "components/form/input";
+import { prependApiUrl } from "utils/api/client";
+import { components } from "schema/main";
+
+const SORT_OPTIONS = [
+  { key: "dataCollectionId", value: "Most Recent" },
+  { key: "globalAlignmentQuality", value: "Alignment Quality" },
+];
+
+interface TomogramListLoaderData {
+  data: components["schemas"]["DataCollectionSummary"][];
+  total: number;
+  limit: number;
+}
+
+const TomogramCard = ({ title, value }: { title: string; value?: string | number | null }) => (
```

Comment:
Since this is just a part of the card for each tomogram, might this be better named as something like `TomogramStat`?